Sources/SourceControl/RepositoryManager.swift

```diff
@@ -9,7 +9,7 @@
 */
 
 import Dispatch
-import class Foundation.OperationQueue
+import Foundation
```

Comment:
Changed this to an unscoped import for simplicity's sake, since this is 5.4 only.